app/helpers/application_helper/button/rbac_user_copy.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::RbacUserCopy < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def disabled?
+    @error_message = _('User [Administrator] can not be copied') if @record.super_admin_user?
```

Comment:
@PanSpagetka I'd better change it here to `Super Administrator can not be copied`...